js/ui/sessionMode.js

```diff
@@ -111,11 +111,11 @@ const _modes = {
                     ['networkAgent', 'polkitAgent', 'telepathyClient',
                      'keyring', 'autorunManager', 'automountManager',
                      'appStore', 'discoveryFeed', 'trayArea',
-                     'updaterManager', 'clubhouse'] :
+                     'updaterManager'] :
```

Comment:
why did you remove this? This makes the Clubhouse button to be hidden always.

Reply:
I removed that because the clubhouse will be launched by the new desktop icon and not by the gnome-shell. But now I think that removing this breaks the clubhouse notifications and more, so I think I need to work on that a bit more to fix that overrides.

Reply:
I've left the clubhouse component and removed all the button code because we won't need that anymore, clubhouse button is not needed anymore.